src/uwtools/cli.py

```diff
@@ -223,16 +223,13 @@ def _dispatch_config_translate(args: Args) -> bool:
 
     :param args: Parsed command-line args.
     """
-    success = True
-    if args[STR.infmt] == FORMAT.atparse and args[STR.outfmt] == FORMAT.jinja2:
-        uwtools.config.atparse_to_jinja2.convert(
-            input_file=args[STR.infile],
-            output_file=args[STR.outfile],
-            dry_run=args[STR.dryrun],
-        )
-    else:
-        success = False
-    return success
```

Comment:
This logic moved into the API.

Reply:
I appreciate the simplicity of moving this logic to the API. It makes this function and the others below broken up into more readable steps.